data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/SimpleCnec.java

```diff
@@ -132,6 +132,10 @@ private boolean isCnecDisconnected(Network network) {
         return !branch.getTerminal1().isConnected() || !branch.getTerminal2().isConnected();
     }
 
+    private boolean isCnecInMainComponent(Network network) {
+        return getTerminal(network).getBusView().getBus().isInMainConnectedComponent();
```

Comment:
I would prefer:
```java
return getTerminal(network).getBusView().getBus().isInMainSynchronousComponent();
```
because usually loadflow and sensitivity computation engines calculate islands based on synchronous components

Reply:
corrected